wordpress-mobile/gutenberg-mobile
# Use Slider from react-native-community lib

Fixes issue with jumping Slider on ios ~13
I added within `gutenberg-mobile` new library `react-native-slider` which is going to replace `Slider` from `react-native` which will be deprecated soon and what's more have a bug on ios ~13.

Ref to gutenberg: https://github.com/WordPress/gutenberg/pull/18806
Ref to WordPress-Android:https://github.com/wordpress-mobile/WordPress-Android/pull/10871
Ref to WordPress-iOS:https://github.com/wordpress-mobile/WordPress-iOS/pull/13019

To test:

I was testing it in WordPress-Android, WordPress-iOS and also within a playground.

1. Add `Spacer` block on mobile
2. Play with `Slider`
  * I suggest you also to set small range eg. minValue=0, maxValue=20 in [RangeCell](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/mobile/bottom-sheet/range-cell.native.js#L181-L182) to check if Slider is not jumping

Update release notes:
